Badges/badges.design-system.mdx

```diff
@@ -1 +1,110 @@
+# 📒 Developer style guide for Badges
 
+The badge component can be used to complement other elements such as buttons or text elements as a label or to show the count of a given data.
+
+Such as the number of comments for an article or how much time has passed by since a comment has been made.
+
+![image](https://user-images.githubusercontent.com/83635276/205807536-98d7ddef-179e-4c91-9ea4-cc053b86d305.png)
+
+## Types of Badges 🔽
+
+1. Default badge
+2. Badges with icon
+3. Notification badge
+4. Button with badge
+5. Badge with icon only
+
+---
+
+## General must known design system guideline :
+
+### Font
+
+For all buttons font is : **`font-medium`**
+
+### Border Radius 🔲
+
+The normal border-radius for the components are `rounded`.
+
+### Hover Effect :
```

Comment:
Remove the hover part as there is no hover effect on badges.

Reply:
Hover effect is used in 2 html file of badges